executor/grant.go

```diff
@@ -487,6 +487,13 @@ func (e *GrantExec) grantDBLevel(priv *ast.PrivElem, user *ast.UserSpec, interna
 		dbName = e.ctx.GetSessionVars().CurrentDB
 	}
 
+	// Attempts to use GRANT ALL as shorthand for granting privileges
+	// at the database leval fail with an error
+	// See https://dev.mysql.com/doc/refman/8.0/en/performance-schema-table-characteristics.html for more detail
+	if dbName == "performance_schema" && priv.Priv == mysql.AllPriv {
```

Comment:
We need to test this doesn't fail with upper cased dbName (see test comment below)

Reply:
`==` has been replaced by `strings.EqualFold`, thank you for your review.